routes/index.js

```diff
@@ -15,5 +15,6 @@ app.use("/trade", require("./trading.js"));
 app.use("/users", require("./users.js"));
 app.use("/wallet", require("./wallets.js"));
 app.use("/featureFlags", require("./featureFlags"));
+app.use("/mehul", require("./testTypescript"));
```

Comment:
Maybe test-typescript would be a better name here?
If you wish You can add a comment as /** author: mehul **/

Reply:
Testing code will be removed before merging to develop